chonk: add nav prompts

JonasBa opened the pull request:
We'll use this to toggle the CTA Tooltip prompts inside navigation

JonasBa committed:
chonk: add nav prompts

JonasBa committed:
chonk: add nav prompts

TkDodo commented on static/app/utils/theme/useChonkPrompt.spec.tsx:
```diff
@@ -0,0 +1,117 @@
+
+import {useChonkPrompt} from './useChonkPrompt';
+
+const queryClient = new QueryClient();
```
note: this will share a `queryClient` between tests. I think it’s better to create a new one for each test in `makeWrapper`.

JonasBa replied:
Uf, yes. Oversight on my end, definitely not desirable to reuse these

JonasBa committed:
chonk: dont reuse queryclient

JonasBa committed:
good guy vscode failing to save again

JonasBa committed:
nav: close dot prompt when help is opened (#89974)

Do not show any indicators if user has opted into chonk already and wire
up dismiss actions with help menu opening

JonasBa committed:
chonk: dismiss all prompts when user clicks help menu

JonasBa committed:
chonk: dont render prompt is isDismissed is undefined

JonasBa committed:
Merge branch 'master' into jb/theme/chonk-cta

JonasBa committed:
fix theme type

JonasBa committed:
noop if both banners have been dismissed or still loading
